### PR TITLE
airportd, automountd, coreautha, coreauthd, nfcd: add page

### PR DESCRIPTION
**Pull request by pixelcmtd**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

**Version of the command being documented (if known):**
Darwin 21.2, macOS 12.1

### CODE REVIEW

**Comment by navarroaxel** [comment]
@pixelcmtd we need links to merge this.

File: pages/osx/automountd.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# automountd
+
+> An automatic mount/unmount daemon for `autofs`. Started on demand by `launchd`.
+> It should not be invoked manually.
```
Comment (navarroaxel) [inline pages/osx/automountd.md:4]:
```suggestion
> It should not be invoked manually.
> More information: <https://www.manpagez.com/man/8/automountd/>.
```

**Comment by pixelcmtd** [comment]
> we need links to merge this.

@navarroaxel why? is there some new rule that i'm not aware of?

**Comment by navarroaxel** [comment]
> @navarroaxel why? is there some new rule that i'm not aware of?

@pixelcmtd We planned to add the [links rule](https://github.com/tldr-pages/tldr/issues/5510#issuecomment-816520051) to the linter but before we want to add all missing links.

**Comment by pixelcmtd** [comment]
> We planned to add the [links rule](https://github.com/tldr-pages/tldr/issues/5510#issuecomment-816520051) to the linter but before we want to add all missing links.

@navarroaxel So half of #6596 is now basically impossible? That helps.

**Review by navarroaxel — APPROVED** [review]

File: pages/osx/coreautha.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# coreautha
+
+> A system agent providing the `LocalAuthentication` framework.
+> It should not be invoked manually.
```
Comment (EmilySeville7cfg) [inline pages/osx/coreautha.md:4]:
```suggestion
> It should not be invoked manually.
> See also: `coreauthd`.
```

File: pages/osx/coreauthd.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# coreauthd
+
+> A system daemon providing the `LocalAuthentication` framework.
+> It should not be invoked manually.
```
Comment (EmilySeville7cfg) [inline pages/osx/coreauthd.md:4]:
```suggestion
> It should not be invoked manually.
> See also: `coreautha`.
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Comment by pixelcmtd** [comment]
> [Preferably one line; two are acceptable if necessary.](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout)

@EmilySeville7cfg

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/osx/coreauthd.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# coreauthd
+
+> A system daemon providing the `LocalAuthentication` framework. See also: `coreautha`.
+> It should not be invoked manually.
```
Comment (navarroaxel) [inline pages/osx/coreauthd.md:4]: We always use the `see also` in the 2nd line
```suggestion
> A system daemon providing the `LocalAuthentication` framework.
> It should not be invoked manually. See also: `coreautha`.
```

File: pages/osx/coreautha.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# coreautha
+
+> A system agent providing the `LocalAuthentication` framework. See also: `coreauthd`.
+> It should not be invoked manually.
```
Comment (navarroaxel) [inline pages/osx/coreautha.md:4]:
```suggestion
> A system agent providing the `LocalAuthentication` framework.
> It should not be invoked manually. See also: `coreauthd`.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by pixelcmtd** [comment]
That's so unlogical in this case tho

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**